docs/_ext/single_sourced_data.py

```diff
@@ -6,22 +6,28 @@
 from copy import copy
 from pathlib import Path
 from re import match
-from typing import Any, Dict, List, Tuple, Union
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Tuple
+from typing import Union
 
 from sphinx.application import Sphinx
 from sphinx.util import logging
 from sphinx.util.docutils import SphinxDirective
-from sphinx.util.nodes import nested_parse_with_titles, nodes
+from sphinx.util.nodes import nested_parse_with_titles
+from sphinx.util.nodes import nodes
 
-# isort: split
 
-from docutils import statemachine
```

Comment:
This is something that is supposed to be close to the sphinx imports (IIRC Sphinx extends/patches docutils so its imports should happen later). It's third-party too, though.

Can we have another split? I don't remember if that would work.

Reply:
@webknjaz have a look, moved it up and added an ignore

Reply:
Okay, looks solved, except that the comment should always have at least two spaces before the start — don't the linters complain about this?

Reply:
flake8 would have had I run it
```
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

docs/_ext/single_sourced_data.py:22:34: E261 at least two spaces before inline comment
from docutils import statemachine # isort: skip
                                 ^
1     E261 at least two spaces before inline comment
1
```
```